library/sensu_check

```diff
@@ -1,81 +1,91 @@
-#!/usr/bin/env ruby
-require 'rubygems'
-require 'antsy'
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
 
-def set_arg_to_bool(var, default = true)
-  if var == 'false' || var == false
-    return false
-  elsif var == 'true' || var == true
-    return true
-  end
-  default
-end
+# Copyright 2014, Blue Box Group, Inc.
+# Copyright 2014, Craig Tracey <craigtracey@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
 
-# defaults
-INTERVAL = 30
+import os
+import traceback
 
-args = Antsy.args
+from hashlib import md5
+from jinja2 import Environment
 
-name         = args[:name]
-use_sudo     = set_arg_to_bool(args[:use_sudo], false)
-auto_resolve = set_arg_to_bool(args[:auto_resolve], true)
-handle       = set_arg_to_bool(args[:handle], true)
-interval     = args[:interval] || INTERVAL
-occurrences  = args[:occurrences] || 2
-plugin       = args[:plugin]
-plugin_args  = args[:args]
-prefix       = args[:prefix]
-env_vars     = args[:env_vars]
+def main():
 
-# parse_env_vars works around Ansible disallowing '='
-# in string arguments, instead environmental variables can be
-# passed in using comma separated string, with a colon supplanting
-# the equals sign
-# e.g. env_vars=PATH:/home/foo,SHELL:/bin/bash
-def parse_env_vars(env_vars)
-  output = []
-  vars = env_vars.split(',')
-  vars.each do |v|
-    output << v.gsub(':','=')
-  end
-  output.join(' ')
-end
+    module = AnsibleModule(
+        argument_spec=dict(
+            name=dict(default=None, required=True),
+            use_sudo=dict(required=False, choices=BOOLEANS, default=False),
```

Comment:
Shouldn't the sudo part come as a task directive, rather than a module argument? Ansible already allows you to set sudo or not on a per-task basis.